recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -120,7 +122,12 @@ def upload_or_check(recipe_dir, owner, channel, variant):
     # The list of built/not skipped distributions
     built_distributions = [(m, path)
                            for m, _, _ in metas
-                           for path in conda_build.api.get_output_file_paths(m)
+                           # TODO: flip this over to .conda when that format
+                           #  is in flight
+                           for path in glob.glob(os.path.join(
+            conda_build.config.croot,
+            'noarch' if m.noarch or m.noarch_python else m.config.host_subdir,
+            '*.tar.bz2'))
```

Comment:
For each output name, you are trying to upload all files.

Reply:
Can you expand on that some? I don't understand if that is what I'm trying to do and doing it incorrectly, or if that is what I should be doing and I'm not

Reply:
What this code did is for each output, calculate the build numbers and the full name(there might be many in rare cases). What the new code is doing is for each output, get the full name of all the outputs.

I think we should get rid of the conda build rendering here and just look at all the packages in `os.path.join(conda_build.config.croot, conda_build.config.subdir)` and `os.path.join(conda_build.config.croot, 'noarch')`.

One issue is that `built_distribution_already_exists` expects a conda-build meta object, but it is used for getting the name and version of the package which can be done by parsing the name as in https://github.com/conda-forge/cf-mark-broken/blob/master/mark_broken.py#L6-L13

Reply:
how do you want to handle the `m.skip` logic? That depends on the rendered meta.yaml

Reply:
Why do we need skip logic at all? Shouldn't conda-build have not produced packages for things that are skipped?